<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraspPlanner</class>
 <widget class="QMainWindow" name="GraspPlanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraspStudio - GraspPlanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>450</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Visualization</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxColModel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>480</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Collision Model</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxCones">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>500</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Friction Cones</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxGrasps">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>520</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show grasps</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonPlan">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Plan grasp(s)</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>181</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>Grasp Planning Options</string>
       </property>
       <widget class="QCheckBox" name="checkBoxFoceClosure">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Force closure</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxGraspNumber">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Number of Grasps</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Timeout, seconds (0=endless)</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxTimeOut">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Min quality</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxQuality">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>400</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>340</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonOpen">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>340</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelInfo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>620</y>
         <width>161</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>Grasps: 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxHighlight">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>540</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Highlight</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
